api/test/e2e/route_with_auth_plugin_test.go

```diff
@@ -17,14 +17,16 @@
 package e2e
 
 import (
+	"fmt"
+	"io/ioutil"
 	"net/http"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 )
 
-func TestRoute_With_Auth_Plugin(t *testing.T) {
+func TestRoute_With_Jwt_Plugin(t *testing.T) {
```

Comment:
the file name should contain `jwt` too

Reply:
The current name is convenient for adding other auth plugin cases here in the future.

Reply:
and I think we should update the file name too

Reply:
fixed.